sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -443,4 +445,111 @@ private PlaySourceInternal getPlaySourceInternalFromRecognizeOptions(CallMediaRe
         }
         return playSourceInternal;
     }
+
+    /**
+     * Send DTMF tones
+     * @param targetParticipant the target participant
+     * @param tones tones to be sent
+     * @param operationContext operationContext (pass null if not applicable)
+     * @return Response for successful sendDtmf request.
+     */
+    public Mono<Void> sendDtmf(CommunicationIdentifier targetParticipant, List<DtmfTone> tones, String operationContext) {
```

Comment:
Remove optional parameter, only keep it in "WithResponse" method.

Reply:
done, thanks!